packages/web-frontend/src/screens/desktop/RootScreen/MainPage.js

```diff
@@ -47,26 +49,33 @@ const MapContainer = styled.div`
   transition: width 0.5s ease;
 `;
 
-const MainPage = ({ enlargedDialogIsVisible, isLoading, sidePanelWidth }) => (
-  <>
-    {/* The order here matters, Map must be added to the DOM body after FlexContainer */}
-    <Container>
-      <EnvBanner />
-      <TopBar />
-      <ContentContainer>
-        <MapDiv />
-        <SidePanel />
-      </ContentContainer>
-      <OverlayDiv />
-      <SessionExpiredDialog />
-      {enlargedDialogIsVisible ? <EnlargedDialog /> : null}
-      <LoadingScreen isLoading={isLoading} />
-    </Container>
-    <MapContainer $rightOffset={sidePanelWidth}>
-      <Map />
-    </MapContainer>
-  </>
-);
+const MainPage = ({ enlargedDialogIsVisible, isLoading, sidePanelWidth }) => {
+  const { isCustomLandingPage } = useCustomLandingPages();
+  if (isCustomLandingPage) {
```

Comment:
I couldn't get the routes in App.js to work with the custom landing page routes as they wouldn't integrate and sync properly with the way we do redux routing so I had to do this slightly clunky way of adding the route which is basically how we handle routes at the moment in the app.